doc/manual.rst

```diff
@@ -566,6 +566,19 @@ The type suffixes are:
   `'f64`             float64
 =================    =========================
 
+If the suffix is not predefined, then the suffix is assumed to be a call
+to a proc, template, macro or other callable identifier that is passed the
+string containing the literal. The identifier needs to contain the apostrophe
+and so will need to be escaped with backticks.
+
+.. code-block:: nim
+  import strutils
+  type nibble = distinct uint8
```

Comment:
you're not using this type; did you intend `'nibble` to return a nibble?

Reply:
changed to `u4` and used distinct type